requirements.txt

```diff
@@ -1,3 +1,2 @@
 apypie>=0.0.5
-ansible>=2.3
```

Comment:
Isn't that still a requirement?

Reply:
yeah. I needed to remove it from the dev file so I can install different ansible versions from the matrix, but the end user requirement is still valid.

Reply:
Can you install the other ansible version beforehand? Than this requirement will already be satisfied and not touched again.